src/sidebar/util/groups.js

```diff
@@ -20,7 +23,48 @@ function combineGroups(userGroups, featuredGroups) {
   featuredGroups.forEach(group => (group.isMember = false));
   userGroups.forEach(group => (group.isMember = true));
 
-  return userGroups.concat(featuredGroups);
+  const groups = userGroups.concat(featuredGroups);
+
+  // Add isSelectable property indicating whether a group can be
+  // annotated in at this particular url.
+  groups.forEach(group => (group.isSelectable = isSelectable(group, uri)));
+
+  // Remove unneeded scopes data from group to save memory.
+  groups.forEach(group => delete group.scopes);
```

Comment:
This seemed like a reasonable thing to do as that info isn't needed but maybe we should just leave it in there?

Reply:
Also from a debugging perspective, if it is convenient to be able to look at `Group` objects in the store and see that their `scopes` property matches `isSelected` and the state of the UI.